spec/unit/validator_spec.rb

```diff
@@ -1313,9 +1313,26 @@ def setup_validator
         validator.stubs(:validate_url)
         validator.validate
 
-        validator.results.map(&:to_s).first.should.match /Dynamic frameworks.*iOS 8.0 and onwards/
+        validator.results.count.should == 2
+        validator.results.map(&:to_s)[0].should.match /`empty\.dylib` does not match the expected static library name format/
```

Comment:
`empty.dylib` is actually a misnamed library so this test had to be updated because it now includes this new warning.